app/src/main/java/org/oppia/android/app/home/HomeViewModel.kt

```diff
@@ -76,10 +76,16 @@ class HomeViewModel(
       promotedActivityListSummaryDataProvider,
       PROFILE_AND_PROMOTED_ACTIVITY_COMBINED_PROVIDER_ID
     ) { profile, promotedActivityList ->
-      listOfNotNull(
-        computeWelcomeViewModel(profile),
-        computePromotedActivityListViewModel(promotedActivityList)
-      )
+      if (profile.numberOfLogins > 1) {
+        listOfNotNull(
+          computeWelcomeViewModel(profile),
+          computePromotedActivityListViewModel(promotedActivityList)
+        )
+      } else {
+        listOfNotNull(
+          computeWelcomeViewModel(profile),
+        )
+      }
```

Comment:
```suggestion
      } else listOfNotNull(computeWelcomeViewModel(profile))
```

Optional, but can be shortened.

Reply:
done